app/assets/json/translations/locale-en.json

```diff
@@ -293,6 +293,8 @@
             "rent_burden_doc_2": "You'll also need to show last month's rent with one of the following:",
             "rent_burden_title": "Rent Burdened Preference",
             "resident_desc": "You'll need one of the following documents showing the address where you live in San Francisco:",
+            "right_to_return_title": "HOPE SF Right to Return Preference",
+            "right_to_return_desc": "You'll need one of the following documents showing your address at Sunnydale Housing Development.",
```

Comment:
```suggestion
```
These aren't being used anymore. Would come out in the grunt translations run I'm about to do anyways